### PR TITLE
[CA-672] Add minnie-kenny into SAM.

### PR DESCRIPTION
**Pull request by yonghaoy**
Ticket: <https://broadworkbench.atlassian.net/browse/CA-672>

minnie kenny: <https://minnie-kenny.readthedocs.io/en/latest/>

I got error saying `git-secrets was not found while git was found` without git-secretes installed

---

**PR checklist**

- [ ] I've followed [the instructions](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/sam/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

### CODE REVIEW

File: project/Testing.scala [hunk 1]
```diff
@@ -32,10 +36,36 @@ object Testing {
...
+  val minnieKenny = inputKey[Unit]("Run minnie-kenny.")
+
   def isIntegrationTest(name: String) = name contains "integrationtest"
 
   lazy val IntegrationTest = config("it") extend Test
 
+  /** Run minnie-kenny only once per sbt invocation. */
+  class MinnieKennySingleRunner() {
+    private val mutex = new Object
+    private var resultOption: Option[Int] = None
```
Comment (yonghaoy) [inline project/Testing.scala:48]: I followed Cromwell and Agora pattern to use a var and change resultOption value. Not sure how to solve the Codacy error. Open to any suggestions.

**Comment by yonghaoy** [comment]
Jenkins retest

File: project/Testing.scala (hunk 1 above)
Comment (dvoet) [reply project/Testing.scala:48]: ignore the codacy error

**Review by dvoet — APPROVED** [review]

File: minnie-kenny.sh [hunk 2]
```diff
@@ -0,0 +1,200 @@
+#!/bin/sh
```
Comment (andy7i) [inline minnie-kenny.sh:1]: it would be nice if we could pull in a master version of this file but that's probably outside the scope for this ticket

**Review by andy7i — APPROVED** [review]
Seems reasonable! I'm not a Scala expert though.

File: minnie-kenny.sh (hunk 2 above)
Comment (yonghaoy) [reply minnie-kenny.sh:1]: I think we want the current version to make sure all eng's new code not adding any secrtes

**Comment by yonghaoy** [comment]
Jenkins retest

**Pull request merged**